docs/preview/index.html

```diff
@@ -56,6 +81,9 @@
     </section>
 
     <section id="section-5">
+      <div style="height:100%;width:100%;display:flex;align-items:center;justify-content:center;">
```

Comment:
Do not use inline styles. It is not always a good practice and it is not readable later in the code. Put it to the class in _section5.scss

Reply:
I used inline styles to easily get rid of this piece of code.
That was temporary button that allows to return to the header section.
Currently the same thing happens when you click on the logo.